code/game/objects/items/storage/backpack.dm

```diff
@@ -62,10 +62,6 @@
 	playsound(src, "rustle", 50, TRUE, -5)
 	qdel(user)
 
-/obj/item/storage/backpack/holding/singularity_act(current_size)
-	var/dist = max((current_size - 2),1)
-	explosion(src.loc,(dist),(dist*2),(dist*4))
-

```

Comment:
I think singilarity BoH explosion problem here. In no qdel on generation of explosion. It can survive on explosion and generate it again.

Reply:
The ensuing explosion seems to wipe them clean. I can tell you that stacking like 10 of them cause the explosions to lag one after another @Iamgoofball

Reply:
that's dennok not goof

Reply:
This is somewhat unrelated to the enormous holes, but still dumb enough that it should be removed. Use clowns and cross your fingers if you wanna contain a singuloose.